src/60_AMP_Ads/AMP_Ads_vs_non-AMP_Ads.html

```diff
@@ -0,0 +1,46 @@
+<!---{
+ "preview": "default",
+ "adSlot": "/30497360/amp_by_example/AMP_Banner_300x250",
+ "skipValidation": "true"
+}--->
+<!--
+  ## Introduction
+
```

Comment:
An AMP ad is an one where the ad is written in AMP HTML. A valid AMP ad does not contain arbitrary JavaScript. All JavaScript in an AMP ad comes from the open source AMP project which is blessed by the AMP project maintainers. As a result, when an ad is created in AMP, it is always guaranteed to do the right thing from a user experience standpoint.